tests/test_core_main.py

```diff
@@ -128,6 +128,29 @@ def test_message_log(caplog):
     assert "temperature: 42.42" in caplog.text, caplog.text
 
 
+@pytest.mark.parametrize("topic", ["test/filter-1"])
+def test_filter_valid_reject(mocker, topic, caplog):
+    """
+    Verify that setting the filteredmessagesloglevel config option changes the log level of the "Filter in section" message.
+    """
+
+    # Bootstrap the core machinery without MQTT.
+    core_bootstrap(configfile=configfile_full)
+
+    # Adjust `filteredmessagesloglevel` configuration setting.
+    mocker.patch("mqttwarn.core.cf.filteredmessagesloglevel", 'DEBUG')
+    mocker.patch("mqttwarn.core.cf.filteredmessagesloglevelnumber", 10)
```

Comment:
I wasn't really sure how to handle this here.  Optimally it seems like we'd set the `filteredmessagesloglevel` string and then this test should cover the config object evaluating that and converting it internally to the `filteredmessagesloglevelnumber` value, but the mock patches here are applied after the configuration object is initialized.  Suggestions?